cl/citations/tests.py

```diff
@@ -37,106 +37,97 @@ def test_make_html_from_plain_text(self) -> None:
         """Can we convert the plain text of an opinion into HTML?"""
         # fmt: off
 
-        full_citation_html = ('<pre class="inline">asdf </pre><span class="'
-                              'citation no-link"><span class="volume">22'
-                              '</span> <span class="reporter">U.S.</span> '
-                              '<span class="page">33</span> </span><pre class='
-                              '"inline">asdf</pre>')
         test_pairs = [
             # Simple example for full citations
-            ('asdf 22 U.S. 33 asdf', full_citation_html),
+            ('asdf 22 U.S. 33 asdf',
+             '<pre class="inline">asdf </pre><span class="'
+             'citation no-link">22 U.S. 33</span><pre class="'
+             'inline"> asdf</pre>'),
 
             # Using a variant format for U.S. (Issue #409)
-            ('asdf 22 U. S. 33 asdf', full_citation_html),
+            ('asdf 22 U. S. 33 asdf',
+             '<pre class="inline">asdf </pre><span class="'
+             'citation no-link">22 U. S. 33</span><pre class="'
+             'inline"> asdf</pre>'),
 
             # Full citation across line break
             ('asdf John v. Doe, 123\nU.S. 456, upholding foo bar',
              '<pre class="inline">asdf John v. Doe, </pre><span class="'
-             'citation no-link"><span class="volume">123</span>\n<span class='
```

Comment:
As you will see, the HTML for all of these is simplified (no more internal spans for volumes, pages, reporters, etc. as I saw no point to them), but the substance of test should hopefully be unchanged.

Reply:
Yep, seems good to me, thanks.